design-reviews/recipes/feature-story-design-review-template.md

```diff
@@ -0,0 +1,60 @@
+# Your Feature or Story Design Title Here
+
+> Does the feature re-use or extend existing patterns / interfaces that have already been established for the project?
+> Does the feature expose new patterns or interfaces that will establish a new standard for new future development?
+
+* [Feature/Story Name](http://link-to-feature-or-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview/Problem Statement
+
+* Describe the feature/story with a high-level summary.
+* Consider additional background and justification, for posterity and historical context.
+
+## Goals/In-Scope
+
+* List the goals that the feature/story will help us achieve.  
```

Comment:
Since for this doc the key goals are the design itself (not necessarily the feature goals but the design review goals), maybe first bullet can be similar to what you have in the task template?

Reply:
Yeah they were similar but added some additional context.